### ISSUE TITLE
32 bit Server: Upload of … exceeds the quota for the folder, when server has plenty of remaining storage

### ISSUE THREAD

**Issue by sgparry** [owncloud/core#40703]
** I have tried my best to format this bug report as requested - if I have messed up anywhere please let me know in the comments; I posted a request for help on central and got no feedback at all **

I have been trying to solve this for several days. After extensive housekeeping, My VPS has 17GB out of 74GB still free, but ownCloud still refuses to sync. I suspect it is something to do with the 32 bitness of my server or the external storage.

### Expected behaviour
Files should be synchronized to server without error

### Actual behaviour
Client complains "There is insufficient space available on the server" and shows "Upload of ... exceeds the quota for the folder" in not synced tab.  Running ```occ files:scan --all``` and ```occ trashbin:cleanup``` makes no difference, nor does retry sync in the client. The error seems to be a conspiracy between the two sides, client and server. The web server logs have shown genuine 507 errors from remote.php, typically on MOVE commands, but also the client seems to retain any error status permanently.

### Steps to reproduce
1. Ensure the server has the following space usage for target user, and is using some in-built storage and some external local storage on the same drive as owncloud's data directory:
```
$ df -h
Filesystem      Size  Used Avail Use% Mounted on
/dev/root        74G   55G   17G  78% /

$ du -d0 -h /var/www/owncloud/data/
17G     /var/www/owncloud/data/
# External local storage 1
$ du -d0 -h /srv/moodle/**redacted**/vle/moodledata/repository/ 
2.6G    /srv/moodle/**redacted**/vle/moodledata/repository/

# External local storage 2
$ du -d0 -h /srv/www/**redacted**/html
124M    /srv/www/**redacted**/html
```
from web interface:
You are using 14.1 GB of 30.1 GB (46.79 %)

2. Install client with non-virtual file mode and selected files.
3. Connect client to user area - choose about 50 - 60 % of the folders to sync.
4. Configure client to synchronize hidden files. Turn off both "Ask for..." confirmations
5. Allow files to sync (if possible).
6. Add one file of 13,496,320 bytes called "%HOME%\ownCloud\pyspace repository\CA-GCE-COMPSCI\assessments\2022-2023\Year13\AP4-Mock-2\paper-2-raw\2.dat".

### Server configuration
**Operating system:** Ubuntu 18.04.6 LTS (32 bit)

**Web server:** apache2  2.4.29-1ubuntu4.27

**Database:** MySQL 5.7.41

**PHP version:** 7.4.33

**ownCloud version:** ownCloud 10.12.0 (stable)

**Updated from an older ownCloud or fresh install:** updated

**Where did you install ownCloud from:**
following apt repo:
deb http://download.opensuse.org/repositories/isv:/ownCloud:/server:/10/Ubuntu_18.04/ /

**Signing status (ownCloud 9.0 and above):**
```
No errors have been found.
```

**The content of config/config.php:** https://gist.github.com/sgparry/fc8528febc5b5bccfd7ed646a3cd4d34

**List of activated apps:**
```
Enabled:
  - activity:
    - Version: 2.7.1
    - Path: /var/www/owncloud/apps/activity
  - calendar:
    - Version: 2.0.0
    - Path: /var/www/owncloud/apps/calendar
  - comments:
    - Version: 0.3.0
    - Path: /var/www/owncloud/apps/comments
  - configreport:
    - Version: 0.2.1
    - Path: /var/www/owncloud/apps/configreport
  - contacts:
    - Version: 1.5.5
    - Path: /var/www/owncloud/apps/contacts
  - dav:
    - Version: 0.7.0
    - Path: /var/www/owncloud/apps/dav
  - federatedfilesharing:
    - Version: 0.5.0
    - Path: /var/www/owncloud/apps/federatedfilesharing
  - federation:
    - Version: 0.1.0
    - Path: /var/www/owncloud/apps/federation
  - files:
    - Version: 1.5.2
    - Path: /var/www/owncloud/apps/files
  - files_external:
    - Version: 0.9.0
    - Path: /var/www/owncloud/apps/files_external
  - files_mediaviewer:
    - Version: 1.0.5
    - Path: /var/www/owncloud/apps/files_mediaviewer
  - files_pdfviewer:
    - Version: 1.0.1
    - Path: /var/www/owncloud/apps/files_pdfviewer
  - files_sharing:
    - Version: 0.14.0
    - Path: /var/www/owncloud/apps/files_sharing
  - files_texteditor:
    - Version: 2.5.0
    - Path: /var/www/owncloud/apps/files_texteditor
  - files_trashbin:
    - Version: 0.9.1
    - Path: /var/www/owncloud/apps/files_trashbin
  - files_versions:
    - Version: 1.3.0
    - Path: /var/www/owncloud/apps/files_versions
  - firstrunwizard:
    - Version: 1.2.0
    - Path: /var/www/owncloud/apps/firstrunwizard
  - gallery:
    - Version: 16.1.2
    - Path: /var/www/owncloud/apps/gallery
  - market:
    - Version: 0.7.0
    - Path: /var/www/owncloud/apps/market
  - notifications:
    - Version: 0.5.4
    - Path: /var/www/owncloud/apps/notifications
  - provisioning_api:
    - Version: 0.5.0
    - Path: /var/www/owncloud/apps/provisioning_api
  - systemtags:
    - Version: 0.3.0
    - Path: /var/www/owncloud/apps/systemtags
  - tasks:
    - Version: 0.9.7
    - Path: /var/www/owncloud/apps/tasks
  - updatenotification:
    - Version: 0.2.1
    - Path: /var/www/owncloud/apps/updatenotification
  - user_ldap:
    - Version: 0.17.0
    - Path: /var/www/owncloud/apps/user_ldap
Disabled:
  - admin_audit:
    - Path: /var/www/owncloud/apps/admin_audit
  - announcementcenter:
    - Path: /var/www/owncloud/apps/announcementcenter
  - customgroups:
    - Path: /var/www/owncloud/apps/customgroups
  - encryption:
    - Path: /var/www/owncloud/apps/encryption
  - enterprise_key:
    - Path: /var/www/owncloud/apps/enterprise_key
  - external:
    - Path: /var/www/owncloud/apps/external
  - files_antivirus:
    - Path: /var/www/owncloud/apps/files_antivirus
  - files_classifier:
    - Path: /var/www/owncloud/apps/files_classifier
  - files_clipboard:
    - Path: /var/www/owncloud/apps/files_clipboard
  - files_external_dropbox:
    - Path: /var/www/owncloud/apps/files_external_dropbox
  - files_external_ftp:
    - Path: /var/www/owncloud/apps/files_external_ftp
  - files_ldap_home:
    - Path: /var/www/owncloud/apps/files_ldap_home
  - files_lifecycle:
    - Path: /var/www/owncloud/apps/files_lifecycle
  - firewall:
    - Path: /var/www/owncloud/apps/firewall
  - graphapi:
    - Path: /var/www/owncloud/apps/graphapi
  - guests:
    - Path: /var/www/owncloud/apps/guests
  - metrics:
    - Path: /var/www/owncloud/apps/metrics
  - oauth2:
    - Path: /var/www/owncloud/apps/oauth2
  - openidconnect:
    - Path: /var/www/owncloud/apps/openidconnect
  - password_policy:
    - Path: /var/www/owncloud/apps/password_policy
  - ransomware_protection:
    - Path: /var/www/owncloud/apps/ransomware_protection
  - sharepoint:
    - Path: /var/www/owncloud/apps/sharepoint
  - systemtags_management:
    - Path: /var/www/owncloud/apps/systemtags_management
  - templateeditor:
    - Path: /var/www/owncloud/apps/templateeditor
  - theme-enterprise:
    - Path: /var/www/owncloud/apps/theme-enterprise
  - user_external:
    - Path: /var/www/owncloud/apps/user_external
  - user_shibboleth:
    - Path: /var/www/owncloud/apps/user_shibboleth
  - web:
    - Path: /var/www/owncloud/apps/web
  - windows_network_drive:
    - Path: /var/www/owncloud/apps/windows_network_drive
  - wopi:
    - Path: /var/www/owncloud/apps/wopi
  - workflow:
    - Path: /var/www/owncloud/apps/workflow
```

**Are you using external storage, if yes which one:** local
```
$ du -d0 -h /srv/moodle/**redacted**/vle/moodledata/repository/
4.8G    /srv/moodle/**redacted**/vle/moodledata/repository/
# du -d0 -h /srv/www/**redacted**/html
124M    /srv/www/**redacted**/html
```

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** LDAP

#### LDAP configuration (delete this part if not used)
https://gist.github.com/sgparry/3f3b2e21a6b231b9537907defe5c0f3c

### Client configuration
**Browser:** Firefox 111.0.1 (64 bit)

**Client version:** ownCloud 3.2.1.10355 [9724d6]

**Operating system:** ```windows-10.0.19044```

OS language: British English

Installation path of client: ```c:\program files\ownCloud```

#### Web server error log
```
https://gist.github.com/sgparry/c1a28276436d68b32dbaccfa5730ded9
```

#### ownCloud log (data/owncloud.log)
```
https://gist.github.com/sgparry/be646093e46e76c922f5149ab8fabb18
```
#### Browser log
```
n/a
```

#### Client logfile:
[ownCloud.zip](https://github.com/owncloud/core/files/11070480/ownCloud.zip)
(sorry, too big to gist)

**Comment by ho4ho** [comment]
- Unrelated by 32bit and duplicate by #40695?
- Related by LDAP: #40693?

**Comment by sgparry** [comment]
Many of #40695 commenters are 32bit...

On 26 March 2023 11:27:10 BST, ho4ho ***@***.***> wrote:
>- Unrelated by 32bit and duplicate by #40695?
>- Related by LDAP: #40693?
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/owncloud/core/issues/40703#issuecomment-1484054349
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by ho4ho** [comment]
Yes, thinking this could close as duplicate by #40695 by all / most report problems by 32bit. Not needing having two issues open reporting by same problem.

**Comment by sgparry** [comment]
Duplicate of #40695

**Comment by sgparry** [comment]
Duplicate of #40695

**Comment by ho4ho** [comment]
Fix by #40709
